VologramsToolkit/Scripts/Player/VolPlayer.cs

```diff
@@ -58,7 +59,7 @@ public class VolPlayer : MonoBehaviour
 
     public bool IsOpen { get; private set; }
     public bool IsPlaying { get; private set; }

```

Comment:
is this 'frame' set in the user interface? i feel like if so i've changed the logic and made a mess of it

Reply:
This just means that external classes can read the current Frame but they can't set it, nothing is broken hwew